vmngclient/api/versions_utils.py

```diff
@@ -138,23 +147,27 @@ def get_device_list_in_installed(self, version_to_set_up: str, devices: List[Dev
         """
         return self._get_device_list_in(version_to_set_up, devices, "installed_versions")
 
-    def get_device_list_in_available(self, version_to_set_up: str, devices: List[Device]) -> List[dict]:
+    def get_device_list_in_available(
```

Comment:
```suggestion
    def get_device_available(
```

is should be good enough too :)

Reply:
Sure, done :)